cfg/pre-commit-config.yaml

```diff
@@ -9,6 +9,14 @@ repos:
     language: 'script'
     name: 'Ensure Shebang'
   repo: 'local'
+-
```

Comment:
How can we ensure that this is run using the proper Poetry spec?

Reply:
I hadn't yet set up the poetry environment for CI yet, but now it's configured in `pre-commit.yaml`

Reply:
I added the poetry environment to the pre-commit hooks and changed the hook `entry` to `poetry run python ./src/python/hooks/src/file-name-conventions.py` so I believe it should run using the poetry env set in the top-level `pyproject.toml`